src/components/common/info/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Avatar, Container, Typography, makeStyles } from '@material-ui/core';
+
+type WebsiteInfoProps = {
+    title: string,
+    favicon: string,
+};
+
+const useStyles = makeStyles(() => ({
+    container: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+    },
+    icon: {
+        marginLeft: 42,
+    },
+    title: {
+        fontSize: 24,
+        fontWeight: 'bold',
+        marginLeft: 18,
+    }
+}));
+
+export default function WebsiteInfo ({ title, favicon }: WebsiteInfoProps) {
+    const classes = useStyles();
+
+    return (
+        <Container className={classes.container}>
+            <Avatar className={classes.icon} alt={title} src={favicon} />
+            <Typography className={classes.title}>
+                {title}
+            </Typography>
+        </Container>
+    );
+};
```

Comment:
An example of appearance.
<img width="439" alt="스크린샷 2020-08-29 오후 10 40 00" src="https://user-images.githubusercontent.com/14817195/91638204-949bba00-ea48-11ea-8ca3-5499d79c438d.png">